### PR TITLE
Sync design changes and fix overlapping graph

### PR DESCRIPTION
**Pull request by JayTailor45**
1. Sync Samyak's design changes with testing branch and remove unwanted code
2. Fix overlapping/ duplicate charts (issue occurred while on print route)

### CODE REVIEW

File: InnoTopicWebsite/src/app/skills/work-experience.component.html [hunk 1]
```diff
@@ -35,72 +35,74 @@ <h1 class="section-title">Overview</h1>
...
-  <div class="language">
-    <div class="app-flags"><app-flags flags="ma"/></div>
-    <label>Arabic</label>
-    <span>Beginner</span>
-  </div>
-  <div class="language">
-    <div class="app-flags"><app-flags flags="in"/></div>
-    <label>Hindi</label>
-    <span>Beginner</span>
+  <div class="languages-grid">
```
Comment (karol-depka) [inline InnoTopicWebsite/src/app/skills/work-experience.component.html:40]: let's move this to separate component, and make it declarative in this format in typescript:
```
{
  English: {
    flags: "gb us",
    level: "Full professional proficiency"
```

**Comment by karol-depka** [comment]
lemme merge soon but I need a nap :D

things look good so far but i need to take another look

**Comment by JayTailor45** [comment]
@karol-depka Now you can test changes on the live site ;)

File: InnoTopicWebsite/src/app/skills/work-experience.component.html (hunk 1 above)
Comment (JayTailor45) [reply InnoTopicWebsite/src/app/skills/work-experience.component.html:40]: @karol-depka I will create a separate task for this. Thanks!

Comment (JayTailor45) [reply InnoTopicWebsite/src/app/skills/work-experience.component.html:40]: @karol-depka I've created a new task for this. https://github.com/InnoTopic/InnoTopic_Website/issues/80

**Comment by JayTailor45** [comment]
@karol-depka Changes of this PR are already merged as a part of https://github.com/InnoTopic/InnoTopic_Website/pull/85 
Please verify and close it.

**Review by karol-depka — APPROVED** [review]
thanks Jay!

**Pull request merged**